### PR TITLE
added sign in and sign up form component, authentication WAI

### PR DESCRIPTION
**Pull request by sophiastan**
I have created a sign-in form and sign-up form component that successfully uses authentication with email/password and Google.

### CODE REVIEW

**Comment by Phoumrint** [comment]
Test: This is the first comment.

File: src/components/sign-in-form/sign-in-form.component.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+} from '../utils/firebase.utils';
+
+import './sign-in-form.styles.scss';
+
+const defaultFormFields = {
+  email: '',
+  password: ''
+}
+
+const SignInForm = () => {
```
Comment (Phoumrint) [inline src/components/sign-in-form/sign-in-form.component.jsx:20]: It looks like you have most of components as functional components. I haven't seen class components.
Normally you create class components if components need to maintain state.
You should also consider creating a class component if the code is more than half a page long.

**Review by Phoumrint — APPROVED** [review]
I just scan through different files instead of reviewing in detail.
In general each component tends to become more complex as you add more features to it.
Consider creating more class components instead of functional components.

**Comment by sophiastan** [comment]
Sign in and sign up component and authentication login is WAI with confirmation

**Pull request merged**